Jigsaw-app/Podfile.lock

```diff
@@ -375,7 +375,7 @@ PODS:
   - PINRemoteImage/PINCache (3.0.1):
     - PINCache (~> 3.0.1)
     - PINRemoteImage/Core
-  - ProgressHUD (13.4)
+  - ProgressHUD (13.5)
```

Comment:
The author removed the old release number, thus I have to upgrade it. It doesn't effect its functionality though.